keyboards/ckeys/handwire_101/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+#include "handwire_101.h"
+#include "action_layer.h"
```

Comment:
`#include QMK_KEYBOARD_H` includes this file automatically, so this line is redundant.

```suggestion
```